app/feed-event-service/feed-event-service.ts

```diff
@@ -1,36 +1,43 @@
 import {FeedEvent} from "../feed-event/feed-event";
 import {Injectable} from "@angular/core";
-import {FEED_EVENTS} from "./feed-event.mocks";
+import {BACKEND_URL} from "../backend-config/backend-config";
+import {AuthHttp} from "../angular2-jwt";
 
 /**
  * Service for fetching and saving FeedEvent instances.
  */
 @Injectable()
 export class FeedEventService {
 
+    private feedEventsUrl: string = `${BACKEND_URL}/feed-events`;
+
+    constructor(
+        private http: AuthHttp
+    ) {
+    }
+
     /**
      * Get a list of FeedEvents.
      * @returns {Promise<Array<FeedEvent>>}
      */
     public list(): Promise<Array<FeedEvent>> {
-        return Promise.resolve(
-            FEED_EVENTS
+        return this.http.get(
+            this.feedEventsUrl
+        ).toPromise().then(
+            (response) => response.json()
+        ).catch(
+            this.handleError
         );
     }
 
     /**
-     * Get a single FeedEvent by ID.
-     * @param id {number} of FeedEvent.
-     * @returns {Promise<FeedEvent>}
+     * Handles any errors communicating with backend.
+     * @param error
+     * @returns {Promise<void>|Promise<T>}
      */
-    public get(
```

Comment:
At least for now, I think list() is the only operation we will have for `FeedEvent`s.

Time permitting, we can look into getting an Amigo-specific list, opening a single event, liking and comments.
(No way we will have time for all that)